components/imageproc/src/format.rs

```diff
@@ -6,12 +6,14 @@ const DEFAULT_Q_JPG: u8 = 75;
 /// Thumbnail image format
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub enum Format {
-    /// JPEG, The `u8` argument is JPEG quality (in percent).
+    /// JPEG, The `u8` argument is JPEG quality (1..100).
     Jpeg(u8),
     /// PNG
     Png,
-    /// WebP, The `u8` argument is WebP quality (in percent), None meaning lossless.
+    /// WebP, The `u8` argument is WebP quality (1..100), None meaning lossless.
     WebP(Option<u8>),
+    /// JPEG XL, The `u8` argument is quality (1..100), None meaning lossless.
+    JXL(Option<u8>),
```

Comment:
Let's call it `JpegXL`, it's going to look less weird than JXL and refer to `jpeg_xl` in the code.

Reply:
Done!